packages/eth/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@audius/eth",
+  "version": "0.0.1",
```

Comment:
Probably want `"private": true` because we don't want to publish this as it's own package right?

Reply:
i think we may actually for people who want to build on top of eth only. wdyt?

Reply:
Oh ok yeah makes sense. Maybe we add the example from your pr description into the readme then?

Reply:
making private for now, but then will make public in a bit!